app/src/main/java/org/mozilla/focus/ext/Bitmap.kt

```diff
@@ -26,3 +31,18 @@ fun Bitmap.arePixelsAllTheSame(): Boolean {
 
     return true
 }
+
+fun Bitmap.withRoundedCorners(cornerRadius: Float): Bitmap? {
```

Comment:
nit: add `@CHeckResult` annotation - it'll make it more clear this Bitmap is not being mutated.